pkg/netflowpolicy/apis/aci.netflow/v1alpha/types.go

```diff
@@ -0,0 +1,63 @@
+package v1alpha
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type PolicyState string
+
+const (
+	Ready  PolicyState = "Ready"
+	Failed PolicyState = "Failed"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// NetflowPolicySpec defines the desired state of NetflowPolicy
+type NetflowPolicySpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	FlowSamplingPolicy NetflowType `json:"flowSamplingPolicy,omitempty"`
+}
+
+//NetflowType
+type NetflowType struct {
+	DstAddr           string `json:"dstAddr"`
+	DstPort           string `json:"dstPort"`
+	Ver               string `json:"type"`
```

Comment:
is it  Ver instead of type ?

Reply:
It is ver. But Tom suggested we use type in the yaml which translates to ver in this code.

Reply:
Since apic only supports netflow and not ipfix, we are exposing "type" in the yaml which internally converts to "ver" = "v5".
netflow = v5, ipfix = v9.